app/controllers/encode_records_controller.rb

```diff
@@ -95,7 +95,13 @@ def progress
     progress_data = {}
     ::ActiveEncode::EncodeRecord.where(id: params[:ids]).each do |encode|
       presenter = EncodePresenter.new(encode)
-      progress_data[encode.id] = { progress: presenter.progress, status: presenter.status }
+      # Set progress = 200 to show the progress bar in UI when failed
+      progress =  if presenter.status.casecmp("failed")
+                    200
```

Comment:
200 seems like a weird choice here.  Also, whatever value we set here should  also be set for failed items on line 78 above.

Reply:
Yeah.. What I wanted to do was to get rid of progress = 1 when failed, and give a value to progress bar >= 100. Because Firefox does not have 2 different ways to style the `progress-bar` and the filled potion (value). In order to show the `failed` status as a red color bar the `<progress>` element should have a value >= 100. Let me see whether there's a better way to this.

Reply:
I think sending 100 is reasonable. Maybe we should consider setting the actual progress value in the encode record to 100 when status goes to failed? @pdinh @cjcolvar?